client_cli/src/main.rs

```diff
@@ -888,6 +915,74 @@ mod asset {
             Ok(())
         }
     }
+
+    #[derive(clap::Args, Debug)]
+    pub struct SetKeyValue {
+        /// AssetId for the Store asset (in form of `asset##account@domain_name')
+        #[clap(long = "asset-id")]
+        pub asset_id: AssetId,
+        /// The key for the store value
+        #[clap(long)]
+        pub key: Name,
+        /// The value to be associated with the key
+        #[clap(long)]
+        pub value: ValueArg,
```

Comment:
This argument should be extensively documented, so that when users see it, they can understand how to pass different values, how they will be interpreted.